back-end/spec/features/streams_a_track_spec.rb

```diff
@@ -1,9 +1,8 @@
-feature 'Tracks the number of plays a song has' do
+ENV['RAILS_ENV']='development'
 
-#defering feature tests until after implemnetation of user story 10: play uploaded music.
+feature 'Tracks the number of plays a song has' do
 
   scenario 'Logged in user plays a song already streamed', js: true do
-    Stream.find_by(track_id: 1).destroy!
```

Comment:
Feature name could be a little bit clearer. Perhaps 'logged in user can play a song that's already streamed'